### DISCUSSION TITLE
Presentable field ignored when inherited into a third table

### DISCUSSION THREAD

**Discussion by R4lfXD** [pocketbase/pocketbase#4494] [Q&A] [answered]
Hey, I don't know if I'm doing something wrong or if this is a bug. I have 3 tables. 
Table 1 is assist table with names, set as Presentable. 
Table 2 has a relation to table 1 to get the name and has it and another fields set as Presentable. 
Table 3 has a relation to Table 2 and should see both Presentable fields from Table 2, but instead it sees correctly only the one that's defined in Table 2, the 'name' that is defined in Table 1 returns an id.

![Table1](https://github.com/pocketbase/pocketbase/assets/31728832/c638db6e-6742-4244-8e0a-b35ee261cc46)
![Table2](https://github.com/pocketbase/pocketbase/assets/31728832/7060536f-4668-4f0f-9163-cbbe14cb01ae)
![Table3](https://github.com/pocketbase/pocketbase/assets/31728832/a1b612cf-7de5-4344-8104-76eac289308a)

**Comment by ganigeorgiev** [comment, marked as answer]
It is not a bug, it just not supported at the moment.

There is an existing issue for this in https://github.com/pocketbase/pocketbase/issues/3666 but remains a low priority as the Admin UI will be [refactored](https://github.com/pocketbase/pocketbase/discussions/4355) anyway.
